From 4eb9c5aecafd8a8861458f389ac16202dcb41699 Mon Sep 17 00:00:00 2001
From: Richard Laager <rlaager@wiktel.com>
Date: Fri, 2 Dec 2016 17:43:52 -0600
Subject: [PATCH 128/268] docs: Wrap long stats output

This way, it does not overflow in the man pages on 80-column terminals.
---
 docs/includes/mon-commands.txt | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/docs/includes/mon-commands.txt b/docs/includes/mon-commands.txt
index e52d04a..e67c7cd 100644
--- a/docs/includes/mon-commands.txt
+++ b/docs/includes/mon-commands.txt
@@ -106,7 +106,8 @@ The event message code and _message_ field are described on the
     generation set named _peerstats_:
 +
 ---------------------------------------------------------------------------------
-48773 10847.650 SPECTRACOM(4) 9714 -0.001605376 0.000000000 0.001424877 0.000958674
+48773 10847.650 SPECTRACOM(4) 9714 -0.001605376 0.000000000
+    0.001424877 0.000958674
 ---------------------------------------------------------------------------------
 +
 [width="100%",cols="<34%,<33%,<33%"]
@@ -139,8 +140,9 @@ show the offset, delay, dispersion and RMS jitter, all in seconds.
 +
 ---------------------------------------------------------------------------------
 5628 54575.160 128.4.1.1 192.168.1.5 3565350574.400229473
-    3565350574.442385200 3565350574.442436000 3565350575.154505763
-    0 4 4 1 8 -21 0.000000 0.000320 PPS 0
+    3565350574.442385200 3565350574.442436000
+    3565350575.154505763 0 4 4 1 8 -21 0.000000 0.000320
+    PPS 0
 ---------------------------------------------------------------------------------
 +
 |==============================================================================
-- 
2.7.4

